app/src/pages/inside/historyPage/historyView/historyTable/historyTable.jsx

```diff
@@ -30,15 +36,27 @@ const messages = defineMessages({
     id: 'HistoryTable.loadMoreHistoryItemsTitle',
     defaultMessage: 'Click here to load more items',
   },
+  itemNamesHeaderTitle: {
+    id: 'HistoryTable.itemNamesHeaderTitle',
+    defaultMessage: 'Name',
+  },
+  launchNumberTitle: {
+    id: 'HistoryTable.launchNumberTitle',
+    defaultMessage: 'Launch #',
+  },
 });
 
 @connect(
   (state) => ({
     items: itemsHistorySelector(state),
     history: historySelector(state),
     visibleItemsCount: visibleItemsCountSelector(state),
+    link: (ownProps) => nameLinkSelector(state, ownProps),
   }),
-  { fetchItemsHistoryAction },
+  {
+    fetchItemsHistoryAction,
+    redirectToTestItem: redirect,
```

Comment:
I think it shouldn't be redirect because we can't use "back" button in this case